src/main/java/frc/robot/constants/drivetrain/TunerConstants.java

```diff
@@ -0,0 +1,128 @@
+import com.ctre.phoenix6.mechanisms.swerve.SwerveModuleConstantsFactory;
+import com.ctre.phoenix6.mechanisms.swerve.SwerveModule.ClosedLoopOutputType;
+import com.ctre.phoenix6.mechanisms.swerve.SwerveModuleConstants.SteerFeedbackType;
+
+import edu.wpi.first.math.util.Units;
+
+public class TunerConstants 
+{
+    // Both sets of gains need to be tuned to your individual robot.
+
+    // The steer motor uses any SwerveModule.SteerRequestType control request with the
+    // output type specified by SwerveModuleConstants.SteerMotorClosedLoopOutput
+    private static final Slot0Configs STEER_GAINS = new Slot0Configs()
+        .withKP(100).withKI(0).withKD(0.05)
+        .withKS(0).withKV(1.5).withKA(0);
+    // When using closed-loop control, the drive motor uses the control
+    // output type specified by SwerveModuleConstants.DriveMotorClosedLoopOutput
+    private static final Slot0Configs DRIVE_GAINS = new Slot0Configs()
+        .withKP(3).withKI(0).withKD(0)
+        .withKS(0).withKV(0).withKA(0);
+
+    // The closed-loop output type to use for the steer motors;
+    // This affects the PID/FF gains for the steer motors
+    private static final ClosedLoopOutputType STEER_CLOSED_LOOP_OUTPUT = ClosedLoopOutputType.Voltage;
+    // The closed-loop output type to use for the drive motors;
+    // This affects the PID/FF gains for the drive motors
+    private static final ClosedLoopOutputType DRIVE_CLOSED_LOOP_OUTPUT = ClosedLoopOutputType.Voltage;
+
+    // The stator current at which the wheels start to slip;
+    // This needs to be tuned to your individual robot
+    private static final double SLIP_CURRENT_A = 300.0;
+
+    // Theoretical free speed (m/s) at 12v applied output;
+    // This needs to be tuned to your individual robot
+    private static final double SPEED_AT_12_VOLTS_MPS = 6.0;
+
+    // Every 1 rotation of the azimuth results in kCoupleRatio drive motor turns;
+    // This may need to be tuned to your individual robot
+    private static final double COUPLE_RATIO = 3.090909090909091;
+
+    private static final double DRIVE_GEAR_RATIO = 7.1328671328671325;
+    private static final double STEER_GEAR_RATIO = 15.42857142857143;
+    private static final double WHEEL_RADIUS_INCHES = 2;
+
+    private static final boolean STEER_MOTOR_REVERSED = true;
+    private static final boolean INVERT_LEFT_SIDE = false;
+    private static final boolean INVERT_RIGHT_SIDE = true;
+
+    private static final String CANBUS_NAME = "";
+    private static final int PIDGEON_ID = 0;
+
+
+    // These are only used for simulation
+    private static final double STEER_INERTIA = 0.00001;
+    private static final double DRIVE_INERTIA = 0.001;
+
+    public static final SwerveDrivetrainConstants DrivetrainConstants = new SwerveDrivetrainConstants()
+            .withPigeon2Id(PIDGEON_ID)
+            .withCANbusName(CANBUS_NAME);
+
+    private static final SwerveModuleConstantsFactory ConstantCreator = new SwerveModuleConstantsFactory()
```

Comment:
screaming Case

Reply:
You got started with changing everything to screaming case but stopped halfway through. Below remove the k from the variables and change to screaming case.